app/views/doc/ext.html.erb

```diff
@@ -34,6 +34,13 @@
             ( <a href="https://backlog.com/git-tutorial/kr/">Korean</a> / <a href="https://backlog.com/git-tutorial/cn/">Simplified Chinese</a> / <a href="https://backlog.com/git-tutorial/tw/">Traditional Chinese</a> / <a href="https://backlog.com/git-tutorial/vn/">Vietnamese</a> )
           </p>
         </li>
+        <li>
```

Comment:
This is being added to the 'Tutorials' section instead of the books section:

![image](https://user-images.githubusercontent.com/1999050/34008832-885d4548-e0fe-11e7-967e-b2be7e2a6ce4.png)


Try running the application before submitting any new patch please. There are instructions under https://github.com/git/git-scm.com#setup

Again, if you need any help let us know